src/traces/barpolar/layout_attributes.js

```diff
@@ -0,0 +1,40 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    barmode: {
+        valType: 'enumerated',
+        values: ['stack', 'overlay'],
+        dflt: 'stack',
+        role: 'info',
+        editType: 'calc',
+        description: [
+            'Determines how bars at the same location coordinate',
+            'are displayed on the graph.',
+            'With *stack*, the bars are stacked on top of one another',
+            'With *overlay*, the bars are plotted over one another,',
+            'you might need to an *opacity* to see multiple bars.'
+        ].join(' ')
+    },
+    bargap: {
+        valType: 'number',
+        // TODO is this correct?
+        // I think 0.2 (like for bars) is a little much, maybe 0.1 would work better?
+        dflt: 0.2,
```

Comment:
Ping. `bar` uses 0.2 as default (unless when [_gapless_](https://github.com/plotly/plotly.js/blob/19f3800bac848e702948fcc32302ded56ea1cdb7/src/traces/bar/layout_defaults.js#L54). I think 0.1 looks before though for `barpolar`. 

![image](https://user-images.githubusercontent.com/6675409/44812263-c3a29b00-aba4-11e8-886b-6e3617b40746.png)

![image](https://user-images.githubusercontent.com/6675409/44812270-cbfad600-aba4-11e8-92e8-17fc09bfd84a.png)


Thoughts?

Reply:
90º bars are unusually large - what value looks best when each bar is only ~15-30º?

Reply:
A few more cases:

at 0.1

![image](https://user-images.githubusercontent.com/6675409/44931072-73efdb00-ad2e-11e8-9ca5-07f89d18eba5.png)

at 0.2

![image](https://user-images.githubusercontent.com/6675409/44931088-7eaa7000-ad2e-11e8-8950-47b4ce7aaf79.png)


---- 

0.1

![image](https://user-images.githubusercontent.com/6675409/44931166-bb766700-ad2e-11e8-8fde-0facd4e7cf03.png)


0.2

![image](https://user-images.githubusercontent.com/6675409/44931178-c16c4800-ad2e-11e8-8029-725d0dbf369a.png)

Reply:
:hammer_and_pick: `0.1`  in -> https://github.com/plotly/plotly.js/pull/2954/commits/6494302c665e1ef18a7c798310f5423fd8f1318e